CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.3-dev
+* Enhancement - Expose two more metadata fields, task definition family and task definition revision [#1295](https://github.com/aws/amazon-ecs-agent/pull/1295)
```

Comment:
Because we have multiple ways to get metadata, can you also specify that this is for 'container metadata file'? Example:`Enhancement - Expose task definition family and task definition revision in container metadata file...`

Reply:
OK, I will change it accordingly.